file_store/src/file_info_poller.rs

```diff
@@ -130,11 +130,19 @@ pub struct FileInfoPollerConfig<Message, State, Store, Parser> {
     p: PhantomData<Message>,
 }
 
+impl<Message, State, Parser>
+    FileInfoPollerConfigBuilder<Message, State, FileStoreInfoPollerStore, Parser>
+{
+    pub fn file_store(self, client: crate::Client, bucket: String) -> Self {
```

Comment:
Might be an opportunity to be nice here and accept anything that can be `.to_string()`.

```rs
pub fn file_store(self, client: crate::Client, bucket: impl Into<String>) -> Self { /* ... */ }
```

This would mean downstream could pass a `&str`, or decide to manage their buckets through an enum.